src/wallet.ts

```diff
@@ -36,49 +43,60 @@ function hashCode(string: string) {
   return hash & 0x7fffffff
 }
 
+/**
+ * 
+ * @ignore
+ */
 function getNodePrivateKey(node: BIP32): string {
   return ecPairToHexString(ECPair.fromPrivateKey(node.privateKey))
 }
 
+/**
+ * 
+ * @ignore
+ */
 function getNodePublicKey(node: BIP32): string {
   return node.publicKey.toString('hex')
 }
 
 /**
- * The BlockstackWallet class manages the hierarchical derivation
- *  paths for a standard blockstack client wallet. This includes paths
- *  for bitcoin payment address, blockstack identity addresses, blockstack
+ *  The `BlockstackWallet` class manages the hierarchical derivation
+ *  paths for a standard Blockstack client wallet. This includes paths
+ *  for Bitcoin payment address, Blockstack identity addresses, Blockstack
  *  application specific addresses.
+ * 
  *  @private
+ * 
+ *  @ignore
```

Comment:
Looks like the whitespacing here is causing the tags to be interpreted as a comments. Should be fixable by just removing the extra column of spaces.

Example: 
![image](https://user-images.githubusercontent.com/1447546/55504521-05099600-561f-11e9-9dc1-bd55316c379a.png)

Reply:
Fixed.